app/src/serializers/teamWithUserRole.serializer.ts

```diff
@@ -0,0 +1,17 @@
+import JSONAPISerializer from "json-api-serializer";
```

Comment:
using [json-api-serializer](https://www.npmjs.com/package/json-api-serializer) instead of [jsonapi-serializer](https://www.npmjs.com/package/jsonapi-serializer) as `json-api-serializer` has typescript types.

Both packages do the same thing but are initialised differently with different options available.